shared/templates/create_permission.py

```diff
@@ -51,7 +51,7 @@ def output_checkfile(target, path_info):
                 "FILEPATH":      full_path,
                 "FILEMODE":      mode,
             },
-            "./output/ansible/file_permissions{0}.sh", path_id
+            "./ansible/file_permissions{0}.sh", path_id
```

Comment:
.yml should be here.

I know that it was here before, but it needs to be fixed, it produces .sh files with ansible content.